Release 0.26.0

salsakran opened the pull request:
Merge changes back into master

sbelak committed:
Remove async fuckup

salsakran committed:
release! [ci all]

salsakran committed:
Merge branch 'release-0.26.0' of github.com:metabase/metabase-init into release-0.26.0

salsakran committed:
Running CI [ci all]

jornh commented on resources/migrations/000_migrations.yaml:
```diff
@@ -3828,88 +3828,4 @@ databaseChangeLog:
       - dropForeignKeyConstraint:
           baseTableName: raw_table
           constraintName: fk_rawtable_ref_database
-          remarks: 'This FK prevents deleting databases even though RAW_TABLE is no longer used. The table is still around to support downgrades, but the FK reference is no longer needed.'
```
Wouldn't a [Liquibase rollback](http://www.liquibase.org/documentation/rollback.html)  be safer (say _someone_ 😊 *blush* already deployed RC2) - or what will otherwise happen if you guys reuse `changeset id: 65` to _someone's_ DATABASECHANGELOG?

salsakran replied:
It's probably most bulletproof if you were to 
`DELETE FROM migrationchangelog WHERE id = '65'`
and drop the tables in question (`computation_job` and `computation_job_result`)

salsakran replied:
actually we talked it out and @senior is going to add a cleanup migration with a precondition. If you can test it for us, that would be awesome!

jornh replied:
Sweet! Yes for sure we’ll do that tomorrow (on CEST time here - so with a bit of luck we should be able to work *with* the sun)

Thanks a bunch guys! Potentially this could save someone other than us too.

iethree committed:
Create migration to will safely revert the async xray tables [ci all]

The `computation_job` and `computation_job_result tables` accidentally
made it into 0.26.0.RC2. This commit will remove those tables as a
separate migration if they are present.

senior committed:
Merge pull request #6041 from metabase/remove-xray-migration

Create migration to will safely revert the async xray tables [ci all]

kdoh committed:
X-ray preview banner (#6051)

* add preview banner

* add icon / survey link

iethree committed:
Reverse the order of dropping the async xray tables

On MySQL dropping the computation_job table first violates the FK
constraints. This commit just flips the order of deletes. It also adds
a new validCheckSum to ensure users that ran the previous version fo
the migration will still work.

Fixes #6050

salsakran committed:
Merge pull request #6052 from metabase/flip-async-xray-table-delete

Reverse the order of dropping the async xray tables
